### ISSUE TITLE
How to get the number of images in the train, val and test set

### ISSUE THREAD

**Issue by AndreaPi** [open-mmlab/mmdetection#6216]
Hi,

related to this issue

https://github.com/open-mmlab/mmdetection/issues/6215

I'm trying to get the size of the train, validation, and test datasets, after MMDetection loads them. I think this script may help me:

https://github.com/open-mmlab/mmdetection/blob/master/tools/misc/browse_dataset.py

Can you help me modify it in order to get what I need? I just need to print the values to screen, I don't need to write a log. Thanks!

**Comment by AndreaPi** [comment]
This seems to work fine:

```
from pathlib import Path
import mmcv
from mmcv import Config
from mmdet.datasets.builder import build_dataset

config_path = Path('/workspace/andrea/instance_segmentation_for_combustor_defects/configs/empty_GT.py')

cfg = Config.fromfile(config_path)

train_dataset = build_dataset(cfg.data.train)
print(train_dataset)
val_dataset = build_dataset(cfg.data.val)
print(val_dataset)
test_dataset = build_dataset(cfg.data.test)
print(test_dataset)
```

All the `print` statements could be substituted by `len` statements, and this would still work. Do you think it's a fine solution, or is there a better method using MMCV & MMDetection?

**Comment by AndreaPi** [comment]
Ok, I didn't get any answers, but the script seems to work fine for me. If there's interest in such a script, I could make a PR and put it in `misc` or `analysis`, I guess - maybe with an option to write to disk or just print to stdout, as it's doing now. Otherwise I'll just close the issue.
